source/common/stats/thread_local_store.h

```diff
@@ -69,11 +69,11 @@ class TlsScope;
 /**
  * Log Linear Histogram implementation that is stored in the main thread.
  */
-class ParentHistogramImpl : public ParentHistogram, public MetricImpl {
+class ThreadLocalParentHistogramImpl : public ParentHistogram, public MetricImpl {
```

Comment:
Renamed. This is more consistent with the other `ThreadLocal` classes here, and it lets me grab the `ParentHistogramImpl` name for my not-thread-local variant.